src/main/java/com/playtika/platforms/test_task/PricingManager.java

```diff
@@ -0,0 +1,59 @@
+public class PricingManager {
+    private static final String PRICE_KEY = "price";
+    private static final String ENV_KEY = "TESTING";
+    private static final PricingManager INSTANCE = new PricingManager();
+
+    private UserManager userManager;
+    private Date actionDate;
+    private PricingLogger prodPricingLogger;
+
+    private PricingManager() {
```

Comment:
I'll prefer make this class more like Util with static methods, instead make it singleton.